packages/jaeger-ui-components/src/TraceTimelineViewer/ReferencesButton.tsx

```diff
@@ -16,10 +16,11 @@ import React from 'react';
 import { css } from 'emotion';
 import NewWindowIcon from '../common/NewWindowIcon';
 import { TraceSpanReference } from '@grafana/data';
-import { UITooltip, UIDropdown, UIMenuItem, UIMenu, TooltipPlacement } from '../uiElementsContext';
+import { UIDropdown, UIMenuItem, UIMenu } from '../uiElementsContext';
 
 import ReferenceLink from '../url/ReferenceLink';
 import { createStyle } from '../Theme';
+import { Tooltip } from '@grafana/ui';
```

Comment:
Why did we change from `UITooltip` to `Tooltip`? The point of using `UITooltip` was to not tie this to grafan/ui in any way, in case we still wanted to make this into a separate project (which may not happen but still). So I wonder if there were some technical issue with the approach of using injected components?

Reply:
Yes I changed it because the tooltip was not showing for me for some reason when used with the wrapper.